workflows/data_pipelines/rne/database/process_rne.py

```diff
@@ -282,8 +283,9 @@ def insert_unites_legales_into_db(list_unites_legales, file_path, db_path):
             indice_repetition,
             complement_localisation,
             distribution_speciale,
+            date_mise_a_jour,
             file_name)
-            VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)
+            VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?,?, ?)
```

Comment:
I would suggest you use code to generate this list if it has as many `?`as there are columns